docs/SlavaUkraini/examples/video-overlay/index.md

```diff
@@ -3,7 +3,7 @@ layout: tutorial_v2
 title: Leaflet on Mobile
 ---
 
-## Video on webpages
+## VideoOverlay - Tutorial
```

Comment:
Each Class (ImageOverlay, VideoOverlay, SVGOverlay) gets a own Headline with two `#`. Something like `## Adding a Image to the map`

Reply:
Thanks, now I understand what is a general idea of this refactoring.